tests/cli/workflow/mip/test_cli_mip_base.py

```diff
@@ -12,13 +12,13 @@
 
 
 def test_spring_decompression_needed_and_started(
-    mocker, cli_runner, caplog, dna_mip_context, case_id
+    mocker, cli_runner, caplog, mip_dna_context, case_id
```

Comment:
Add type hints. It will make it easier to understand what the test needs as input